src/load-test/prod/networking.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_virtual_network" "weu_load_test" {
+  name                = "${local.project}-load-test-vnet-01"
+  resource_group_name = azurerm_resource_group.load_test.name
+  location            = azurerm_resource_group.load_test.location
+  address_space       = ["10.40.0.0/22"]
+
+  tags = local.tags
+}
+
+resource "azurerm_subnet" "weu_load_test" {
+  name                 = "${local.project}-load-test-snet-01"
+  resource_group_name  = azurerm_resource_group.load_test.name
+  virtual_network_name = azurerm_virtual_network.weu_load_test.name
+  address_prefixes     = ["10.40.0.0/27"]
```

Comment:
I was wondering whether we could use a larger CIDR for this subnet. Did you oversee any other subnets that may be useful in the future to cover the VNet CIDR?

Reply:
Only if multiple tests are running at the same time, I can't see any other scenario
I tried to keep it small, but I don't see too much difference among this and a slight bigger subnet